homework01/artur/vector.h

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+class Vector
+{
+ public:
+  Vector();
+  Vector(const int, const int);
+  Vector(const Vector&);
+  
+  const int GetX() const;
```

Comment:
Good to mark the method `const`, although there is no added value regarding the returned value, and in general this is not recommended.

Reply:
I am using those getters in other functions, that take const Vector& as an argument. Without those, I was getting "argument discards qualifiers [-fpermissive]" compile error

Reply:
Yes &mdash; you need the last `const`, otherwise you will get this compiler error when using the method in other non-const method. The first `const` though, i.e. the one of the returned value, does not change anything in this case.